packages/lambda-xyz/src/routes/tile.ts

```diff
@@ -40,6 +40,10 @@ const LoadingQueue = pLimit(Env.getNumber(Env.TiffConcurrency, 5));
 /** Background color of tiles where the tileset does not define a color */
 const DefaultBackground = { r: 0, g: 0, b: 0, alpha: 0 };
 
+function notFound(): LambdaHttpResponse {
+    return new LambdaHttpResponse(404, 'Not Found');
```

Comment:
is there anything stopping us doing
```
const NotFound = new LambdaHttpResponse(404, 'Not Found');
```

I dont think we need new instance of this every time.